examples/webgpu_postprocessing_ao_prepass.html

```diff
@@ -0,0 +1,244 @@
+			import { ao } from 'three/addons/tsl/display/GTAONode.js';
+			import { traa } from 'three/addons/tsl/display/TRAANode.js';
+
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+			import { DRACOLoader } from 'three/addons/loaders/DRACOLoader.js';
+			import { RoomEnvironment } from 'three/addons/environments/RoomEnvironment.js';
+			import { GLTFLoader } from 'three/addons/loaders/GLTFLoader.js';
+
+			import { Inspector } from 'three/addons/inspector/Inspector.js';
+
+			let camera, scene, renderer, postProcessing, controls;
+
+			let aoPass, traaPass;
+
+			const params = {
+				samples: 16,
+				distanceExponent: 1,
+				distanceFallOff: 1,
+				radius: 0.25,
+				scale: 1,
+				thickness: 1,
+				aoOnly: false
+			};
+
+			init();
+
+			async function init() {
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.1, 50 );
+				camera.position.set( 1, 1.3, 5 );
+
+				scene = new THREE.Scene();
+
+				renderer = new THREE.WebGPURenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setAnimationLoop( animate );
+				renderer.inspector = new Inspector();
+				document.body.appendChild( renderer.domElement );
+
+				await renderer.init();
+
+				const environment = new RoomEnvironment();
+				const pmremGenerator = new THREE.PMREMGenerator( renderer );
+
+				scene.background = new THREE.Color( 0x666666 );
+				scene.environment = pmremGenerator.fromScene( environment ).texture;
+				environment.dispose();
+				pmremGenerator.dispose();
+
+				//
+
+				controls = new OrbitControls( camera, renderer.domElement );
+				controls.target.set( 0, 0.5, - 1 );
+				controls.update();
+				controls.enablePan = false;
+				controls.enableDamping = true;
+				controls.minDistance = 2;
+				controls.maxDistance = 8;
+
+				//
+
+				postProcessing = new THREE.PostProcessing( renderer );
+
+				//
+
+				const prePass = pass( scene, camera ).toInspector( 'Normal', ( inspectNode ) => colorSpaceToWorking( inspectNode, THREE.SRGBColorSpace ) );
+				prePass.name = 'Pre-Pass';
+				prePass.setMRT( mrt( {
+					output: directionToColor( normalView ),
+					velocity: velocity
+				} ) );
+				prePass.transparent = false;
+
+				const prePassNormal = sample( ( uv ) => {
+
+					return colorToDirection( prePass.getTextureNode().sample( uv ) );
+
+				} );
+
+				// bandwidth optimization
+
+				const normalTexture = prePass.getTexture( 'output' );
+				normalTexture.type = THREE.UnsignedByteType;
+
+				//
+
+				const scenePass = pass( scene, camera ).toInspector( 'Color' );
+				scenePass.name = 'Scene Pass';
+
+				const prePassDepth = prePass.getTextureNode( 'depth' ).toInspector( 'Depth', () => {
+
+					return prePass.getLinearDepthNode();
+
+				} );
+				const scenePassVelocity = prePass.getTextureNode( 'velocity' ).toInspector( 'Velocity' );
```

Comment:
Can you rename this to `prePassVelocity` and move it to `prePassNormal` block as well?